src/telemetry.ts

```diff
@@ -47,7 +47,12 @@ export interface NewComponentCommandProps {
     'use_existing_devfile': boolean;
 }
 
-export type TelemetryProps = Partial<CommonCommandProps & NewComponentCommandProps>;
+export interface copyDevFileProps {
```

Comment:
No need for new properties, just use component_type above to report devfile name.